### PR TITLE
[OutlookTasks] Open-sourcing the OutlookTasks connector

### PR DESCRIPTION
**Pull request by joem-msft**
Adding the OutlookTasks connector with operations that are safe to open-source.

### CODE REVIEW

File: certified-connectors/OutlookTasks/apiProperties.json [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+    "iconBrandColor": "#0072c6",
+    "capabilities": [
+      "actions"
+    ],
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2generic",
+          "clientId": "99da6bf4-4525-4bfa-b39f-b6a35789c240",
```
Comment (mamurshe) [inline certified-connectors/OutlookTasks/apiProperties.json]: Can you please remove the client id?

File: certified-connectors/OutlookTasks/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,481 @@
...
+        }
+      }
+    },
+    "/actions/CreateTaskFolder/v2/me/taskfolders": {
+      "post": {
+        "tags": [
+          "Tasks"
+        ],
+        "summary": "Create a task folder",
+        "description": "This operation creates a new task folder in the default task group (`My Tasks`) in your mailbox.",
```
Comment (mamurshe) [inline certified-connectors/OutlookTasks/apiDefinition.swagger.json]: Should `My Tasks` be the single quotes 'My Tasks'?

**Comment by mamurshe** [comment]
Please remove the icon and add a readme.md.

File: certified-connectors/OutlookTasks/apiDefinition.swagger.json (hunk 2 above)
Comment (joem-msft) [reply certified-connectors/OutlookTasks/apiDefinition.swagger.json]: `` are MD syntax, which is what's supported in these description properties. Note how even your comment got the special syntax highlighting.

**Pull request merged**